src/main/java/uk/gov/hmcts/reform/coh/domain/QuestionState.java

```diff
@@ -1,20 +1,27 @@
 package uk.gov.hmcts.reform.coh.domain;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import javax.persistence.*;
 import java.util.Objects;
 
-@Entity(name = "Question State")
+@Entity
 @Table(name = "question_state")
 public class QuestionState {
 
     public static final int DRAFTED = 1;
-    public static final int ISSUED = 2;
+    public static final int SUBMITTED = 2;
+    public static final int ISSUED = 3;
+
```

Comment:
These should not be hard-coded. Either put it in an Enum or call the the QuestionStateService to get it.

Fine to do this in another branch